chain/chain/src/store_validator.rs

```diff
@@ -249,13 +249,6 @@ impl StoreValidator {
                 DBCol::StateDlInfos => {
                     let block_hash = CryptoHash::try_from(key_ref)?;
                     let state_sync_info = StateSyncInfo::try_from_slice(value_ref)?;
-                    // StateSyncInfo is valid
-                    self.check(
```

Comment:
Why did you remove this check?

Reply:
Also see the comment above about avoiding the DB migration